tools/handshake-runner/handshake-runner.cpp

```diff
@@ -209,28 +211,28 @@ void executeOp(mlir::StoreOp op, std::vector<Any> &in, std::vector<Any> &out,
   ref[address] = in[0];
 }
 
-void executeOp(mlir::handshake::ForkOp op, std::vector<Any> &in,
+void executeOp(circt::handshake::ForkOp op, std::vector<Any> &in,
```

Comment:
Why are all these explicit namespace qualifiers required?  Please drop them to reduce clutter.